server/seed/index.js

```diff
@@ -0,0 +1,51 @@
+require("dotenv").config();
+const colors = require("colors");
+const User = require("../models/User");
+const Profile = require("../models/Profile");
+const { userArray, profileArray } = require("./sampleProfiles");
+const connectDB = require("../db");
+
+connectDB();
+
+console.log("loading mock users");
```

Comment:
Actually, the `seed/index.js` is not a part of the main application, I included it because it provides an easy way to programmatically load the database with mock data, for the purpose of testing, I included this in the description of the PR. The console logs were intentionally left to aid any of my colleagues who intends to use this feature to seed the database. Do I still go ahead and remove all console logs here?